src/driver/amdxdna/aie2_solver.h

```diff
@@ -66,41 +68,20 @@ struct xrs_action_load {
 	struct aie_part         part;
 };
 
-/*
- * Define the power level available
- *
- * POWER_LEVEL_MIN:
- *     Lowest power level. Usually set when all actions are unloaded.
- *
- * POWER_LEVEL_n
- *     Power levels 0 - n, is a step increase in system frequencies
- */
-enum power_level {
-	POWER_LEVEL_MIN = 0x0,
-	POWER_LEVEL_0   = 0x1,
-	POWER_LEVEL_1   = 0x2,
-	POWER_LEVEL_2   = 0x3,
-	POWER_LEVEL_3   = 0x4,
-	POWER_LEVEL_4   = 0x5,
-	POWER_LEVEL_5   = 0x6,
-	POWER_LEVEL_6   = 0x7,
-	POWER_LEVEL_7   = 0x8,
-	POWER_LEVEL_NUM,
-};
```

Comment:
This enum has been moved to aie2_pci.h, which make sense to me.
I aware of that the POWER_LEVEL_MIN was removed in the new definition. Is this expected?

Reply:
I guess we don't need Power_level_min, power_level_min and power_level_0 are same. so I removed it.